sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ProtoDescriptorGenerator.scala

```diff
@@ -43,7 +44,11 @@ object ProtoDescriptorGenerator {
 
     protoBuilder.addDependency("google/protobuf/any.proto")
     protoBuilder.addService(service)
-    messages.foreach(protoBuilder.addMessageType)
+    messages
+      .filterNot { desc =>
+        desc.getName == JavaPbAny.getDescriptor.getFullName
+      }
```

Comment:
Leftover, I will remove. We can't filter it in a reliable way. 

This is filtered already by using `None`, btw